js/apps/admin-ui/src/user/components/SelectComponent.tsx

```diff
@@ -86,7 +86,13 @@ export const SelectComponent = ({
                 key={option}
                 value={option}
               >
-                {option}
+                {t(
+                  unWrap(
+                    (attribute.annotations?.["inputOptionLabels"] as any)?.[
```

Comment:
Why does this need to be converted to `any`?

Reply:
because annotations can be anything according to the java model